src/components/memoBlock/memo.component.tsx

```diff
@@ -76,13 +92,38 @@ function MemoBlock(props: Props): JSX.Element {
       }, 1000);
     }
   }
+
+  const levels = ["3*4", "4*4", "4*5", "4*6", "4*7", "4*8"];
```

Comment:
Its value never changes so it can be placed at the module level. This way, you don't increase the component size, thus gaining readability points when reading the component content. 

```suggestion
```